changelogs/unreleased/bump-docstring-parser.yml

```diff
@@ -0,0 +1,4 @@
+description: Bump docstring-parser dependency and account for non-backwards compatible changes.
+change-type: patch
+destination-branches:
+  - master
```

Comment:
Should we include the message in the bugfix section? Otherwise we will have a release without a changelog entry on the documentation pages.

Reply:
Good idea, I'll add that.